Ansible/inventories/inventory.za-chpc

```diff
@@ -129,7 +129,8 @@ worker-nodes
 ############## Service oriented grouping ##############
 # (here comes the magical power of ansible... Group things
 # in as much as you can)
-
+[EOS]
+storage-elements
```

Comment:
Not sure how dashes are handled in group names. Please check.

Reply:
other places have dashes so if it used to work it works now?